src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+body {
+  margin: 0;
+}
+
+.card {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__product-name {
+    margin-top: 40px;
+    margin-bottom: 0;
+    font-family: $font-family;
+    color: $main-accent-color;
+    width: 100%;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__product-id {
+    margin-top: 4px;
+    margin-bottom: 0;
+    font-family: $font-family;
+    color: $secondary-accent-color;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price-value {
+    margin: 0;
+    font-family: $font-family;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: $secondary-accent-color;
+  }
+
+  &__price-total {
+    margin: 0;
+    font-family: $font-family;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: $main-accent-color;
+  }
+
+  &__button {
+    display: flex;
+    margin-top: 16px;
+    border: 1px solid $blue-accent-color;
+    background-color: $blue-accent-color;
+    color: $white-accent-color;
+    border-radius: 5px;
```

Comment:
The variable `$white-accent-color` should be defined in your `veriables.scss` file or imported correctly if it's defined elsewhere. Ensure that `$white-accent-color` is available in this context.